### PR TITLE
1 user entity

### LINKED ISSUE #1: User-entity
Add user entity

### PR DESCRIPTION
**Pull request by Win-ther**
Closes #1 

- Adding User-entity with columns:
  - id - Autogenerated
  - userName - which is unique
  - image - which is a bytearray in preparation for implementation of adding a profilepicture
  - name
  - email

With getters/setters and equals/hashCode.

### CODE REVIEW

File: src/main/java/se/iths/SpringbootGroupProject/entities/User.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+@Entity
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(unique = true)
+    private String userName;
+    @Lob
+    private Byte[] image;
+    private String name;
```
Comment (Frlundsten) [inline src/main/java/se/iths/SpringbootGroupProject/entities/User.java]:
```suggestion
    private String firstName;
    private String lastName;
```

or a method to combine the name

setName(String first, String last)
name = first + " " + last;

**Review by Frlundsten — APPROVED** [review]
Looks good. Well done 👍

**Review by altmannk — APPROVED** [review]
LGTM 🫡

**Review by marcusju23 — APPROVED** [review]
👍

**Pull request merged**